src/pages/Groups/Group1/Screen2.tsx

```diff
@@ -1,7 +1,134 @@
 import React from 'react'
+import LeftOutlined from '@ant-design/icons/lib/icons/LeftOutlined'
+import UnorderedListOutlined from '@ant-design/icons/lib/icons/UnorderedListOutlined'
+import {
+  WeekContainer,
+  Time_container,
+  Timing_container,
+  Bookings_container,
+  IndividualBookings,
+  TopNav,
+  Header,
+  EventText,
+  EventTextWeekly,
+  TimeText,
+  LocationText,
+  LocationTextWeekly,
+  BotNav,
+  StyledButton,
+  ButtonContainer,
+  ToggleButton,
+  Calendar_container,
+  Day_container,
+  Days,
+  EventCard,
+} from './styles/Screen2.styled'
+import homeIconUnselected from '../../../assets/Group1/homeIconUnselected.svg'
+import facilitiesIconUnselected from '../../../assets/Group1/facilitiesIconUnselected.svg'
+import calenderIconSelected from '../../../assets/Group1/calenderIconSelected.svg'
+import profileIconUnselected from '../../../assets/Group1/profileIconUnselected.svg'
+import { useState } from 'react'
 
 const Screen2 = () => {
-  return <>this is screen 2</>
+  const [isDailyMode, setDailyMode] = useState(false)
+
+  return (
+    <>
+      {renderTopNav(isDailyMode)}
```

Comment:
1. "render" is a verb, but what it does is not an action, so try to name it aligned to its purpose. 
it returns a component, so it is safe to name this method as a "TopNav".


2. TopNav a little too vague, try calling it Screen2TopNav <- since other places have a top nav too

Reply:
fixed in commit "b633ef9099261dc56d797b8bc08d096769891e48"